doc/src/bond_quartic.rst

```diff
@@ -1,45 +1,92 @@
-.. index:: bond_style quartic
-.. index:: bond_style quartic/omp
+.. index:: bond_style quartic/breakable
+.. index:: bond_style quartic/breakable/omp
+.. index:: bond_style quartic/unbreakable
 
 bond_style quartic command
 ==========================
 
 Accelerator Variants: *quartic/omp*
 
+bond_style quartic/breakable command
+====================================
+
+Accelerator Variants: *quartic/breakable/omp*
+
+bond_style quartic/unbreakable command
+======================================
+
 Syntax
 """"""
 
 .. code-block:: LAMMPS
 
-   bond_style quartic
+   bond_style style
+
+* style = *quartic/breakable* or *quartic/unbreakable*
 
 Examples
 """"""""
 
 .. code-block:: LAMMPS
 
-   bond_style quartic
+   bond_style quartic/breakable
    bond_coeff 2 1200 -0.55 0.25 1.3 34.6878
 
+   bond_style quartic/unbreakable
+   bond_coeff 5 200  -1.55 0.57 0.7  3.022
+   bond_coeff 6 2.0  -1.05 0.27 0.6  1.022
+
 Description
 """""""""""
 
-The *quartic* bond style uses the potential
+The *quartic/breakable* and *quartic/unbreakable* bond styles use the potential
 
 .. math::
 
    E      & = E_q + E_{LJ} \\
-   E_q    & = K (r - R_c)^ 2 (r - R_c - B_1) (r - R_c - B_2) + U_0 \\
+   E_q    & = \left\{ \begin{array} {l@{\quad:\quad}l}
```

Comment:
It would be cleaner to have two `.. math::` blocks. One for each variant.

Reply:
Updated.